web-service-consumer/1.6/modules/ROOT/pages/edits/index.adoc

```diff
@@ -0,0 +1,76 @@
+= Web Service Consumer Connector 1.6 - Mule 4
+:page-aliases: connectors::web-service/web-service-consumer.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Web Service Consumer (Web Service Consumer Connector) consumes a SOAP web service from a Mule app, to acquire data from an external source. In many cases, you can use Anypoint Connector for Workday (Workday Connector) or Anypoint Connector for ServiceNow (ServiceNow Connector) to connect to a service provider, however; when no connector is available for a specific web service, the easiest way to consume the service is to use the Web Service Consumer Connector.
+
+The main features of the connector include:
+
+* Consume DOC literal web services
+* Use SOAP multipart messages
+* Use SOAP headers
+* Support DataSense for SOAP headers, SOAP body, and attachments
+* Use embedded DataWeave transformations inside the operation
+* Support an unified experience for SOAP with attachments and MTOM handling
+* Custom HTTP configuration as a transport
+* Support Web Service Security (WS Security)
+
+[NOTE]
+Web Service Consumer Connector doesn't support RPC WSDLs.
+
+For compatibility information and fixed issues, see the xref:release-notes::connector/connector-wsc.adoc[Web Service Consumer Release Notes]
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* SOAP, WSDLs
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements in a Mule flow
+* Global elements
+* DataWeave
+* Understand how to create a Mule app using Design Center or Anypoint Studio
+
+Before creating an app, you must have access to Anypoint Studio.
+
+//== Common Use Cases For the Connector
+
+// Add a lead in sentence and then list common use cases for the connector
+
+// For examples, see xref:<connector-name>-connector-examples.adoc[Examples].
+
+== Audience
+
+* New user
++
+To create your Mule app, see:
+
+* xref:web-service-consumer-studio.adoc[Studio Configuration]
+* xref:web-service-consumer-config-topics.adoc[Additional Configuration Information]
+to help extend your understanding of the Web Service Consumer Connector
```

Comment:
Is this really to extend knowledge of the connector or is it about customization?